### ISSUE TITLE
Have the authors considered the following issues?

### ISSUE THREAD

**Issue by UESTC-Med424-JYX** [wsntxxn/TextToAudioGrounding#3]
Thanks to the author for his excellent work, but I have some doubts in the experimental part. The author used AudioCaps for training (which includes validation and test sets), and re-split the test set according to the audio subtitles, which inevitably hindered the subsequent work. Because other methods such as PicoAudio hope to use WSTAG for subsequent work, and WSTAG using the AudioCaps test set for training inevitably affects the subsequent work. This only represents my personal doubts and concerns. May I ask if the author has considered such issues?

**Comment by UESTC-Med424-JYX** [comment]
In addition, there is an excellent timestamp dataset AudioSet-Strong. Why didn't the author consider using this dataset for training?

**Comment by wsntxxn** [comment]
1. About the split 
* In the initial release of grounding data, we used the original split of AudioCaps train/val/test split. In the following experiments we found the test split contains much longer sentences. We are worried about the influence of distribution discrepancy on the validity of test set so we re-split it (because we observe that the trend on val during training does not correspond with the test set). 
* Since WSTAG only serves as data pre-processing models to filter out unreliable data, I guess the use of AudioCaps does not have a big influence, as the original audio-text data are not used in TTA training. However, it is a really good suggestion to provide a checkpoint that AudioCaps test data is excluded in training. Thanks!
2. In fact, we did use AudioSet-Strong to pre-train the audio encoder. But you are right, it can also be used to train the whole grounding model to further enhance its performance. Since the strong labelling of AudioSet is labor-extensive, we design experiments to validate the effect of weakly-supervised learning, which is more scalable.
